tmail-backend/mailets/src/test/java/com/linagora/tmail/mailet/IndexContactsTest.java

```diff
@@ -239,6 +239,27 @@ void serviceShouldNotClearAttributeValueAfterConsumption() throws MessagingExcep
             .isEqualTo(attributeValue.value());
     }
 
+    @Test
+    void serviceShouldDispatchEventWhenEmailAttributeHasMailboxName() throws MessagingException {
+        mailet.init(MAILET_CONFIG);
+
+        FakeMail mail = FakeMail.builder()
+            .name("mail1")
+            .attribute(ATTRIBUTE_NAME.withValue(AttributeValue.of(
+                    """
+                        {
+                            "userEmail": "%s",
+                            "emails": ["%s"]
+                        }
+                        """.formatted(SENDER, "Name1 <" + RECIPIENT + ">")
```

Comment:
Why did we end up with `Name <RCPT...>` in there in the first place? Can we please investigate?

I am ok for downstream sanitation but we also need to identify and fix the source of the issue: why we have an attribute like that in the first place.

And here it is clearly not done!

Reply:
This is logic of [ContactExtractor#getRecipients](https://github.com/apache/james-project/blob/master/mailet/standard/src/main/java/org/apache/james/transport/mailets/ContactExtractor.java#L134)

(since 2017)

And we already has the test case relate for this format

https://github.com/apache/james-project/blob/master/mailet/standard/src/test/java/org/apache/james/transport/mailets/ContactExtractorTest.java#L317